x/nexus/types/params.go

```diff
@@ -23,7 +29,10 @@ func KeyTable() params.KeyTable {
 // DefaultParams creates the default genesis parameters
 func DefaultParams() Params {
 	return Params{
-		ChainActivationThreshold: utils.NewThreshold(55, 100),
+		ChainActivationThreshold:              utils.NewThreshold(55, 100),
+		ChainMaintainerMissingVoteThreshold:   utils.NewThreshold(20, 100),
+		ChainMaintainerIncorrectVoteThreshold: utils.NewThreshold(3, 100),
+		ChainMaintainerCheckWindow:            100,
```

Comment:
@sergeynog @milapsheth Let me know your thinking on these params. Right now, it's saying that if a validator voted incorrectly for >3 or missed voting for >20 of the past 100 polls, then they will be deregistered.

Reply:
I think we should increase the window to something like 5000 or 10000 blocks with a tighter incorrect vote threshold, `1%`?. If we start processing more polls per block, say 100, that's only about 50 blocks, where it's feasible that temporal issues in some RPC nodes on an external chain could cause larger downtimes on our network due to more unregistered maintainers. For e.g. if there's a DOS attack on an external chain, that could result in some RPC nodes being unresponsive for a while but it's not that they're voting incorrectly, so we don't want to penalize them too for a problem they aren't responsible for. It also gives validators some time to respond. For context, we've increased the downtime for Tendermint to ~1 day due to a similar argument.

Reply:
This is per chain though. And of course it is them to blame. Polls can only be triggered from txs on axelar-netowrk and it's the validators' job to find some EVM nodes that can handle more RPC calls than the number of txs/msgs we can handle.

Reply:
My point was different, we've seen some external chains, like Polygon, suffer intermittent throughput issues, where it was an issue with the chain, and not the maintainers of the RPC nodes. So, for e.g. if Polygon has intermittent issues for a couple of hours which causes RPC nodes to be partly unresponsive for a bunch of validators, we'll be deregistering them, and cause even more downtime until the validators have re-registered. Similarly, a chain could be under a DOS attack that could affect RPC nodes and but since we deregister validators, our downtime would extend beyond the DOS attack until validators have had a chance to re-register. In general, we want to give them some time to correct the issue or not have to worry about it if it was temporary (couple of hours say).

Reply:
Also, the window isn't about block, it's about the number of polls of that chain concerned.

Reply:
Still need to revisit these params, but don't wanna block the other PR

Reply:
I think if there are issues with the connected network, the response should be to pause transfers to/from it via governance [or instruct validators to deregister themselves], not to auto-deregister. The reason is that at the moment, I think it'd be hard to predict all on-chain rules that we'd need to code to deal with major chain problems. So, I'd give validators more room for errors when their individual nodes are down. Otherwise, if the rule is triggered often due to "normal" temporary, it'll cause too much inconvenience. 
how about: 500 pools; validator is kicked out if they voted on 25 incorrectly or missed voting for 100 ? 
these parameters are upgradable with an on-chain param upgrade vote, right?

Reply:
I'm good with those numbers, and yes these params are upgradable on-chain as usual.